### DISCUSSION TITLE
Which companies are using Unpoly?

### DISCUSSION THREAD

**Discussion by triskweline** [unpoly/unpoly#666] [General]
We [want](https://github.com/unpoly/unpoly/discussions/664) to compile a list of companies and projects that use Unpoly. Maybe we can add some logos to [unpoly.com](https://unpoly.com/) to show that Unpoly is a stable solution that has seen a lot of production traffic.

If your company is using Unpoly for itself, or for a client, **please add the name for the project or company below.**

> **Tip**: If you're unsure if a screen is built with Unpoly, open the developer consol (`CTRL+Shift+J`) and see if  `window.up` is defined.

I'm starting with some apps that I know of:

International brands
--------------------

- [Bosch](https://www.bosch-betterfood.com/)
- [TUI fly](https://claims.tuifly.com)
- [Audi](https://www.audi-mediacenter.com/)
- [Volkswagen](https://www.volkswagen-newsroom.com/)
- [Steady](https://steadyhq.com/)
- Siemens (internal app)

Open source projects
--------------------

- [AdonisJS](https://adonisjs.com/)
- [Adocasts](https://adocasts.com/)
- [classroombookings](https://www.classroombookings.com/)
- [Agorakit](https://www.agorakit.org/en/)

Companies in Germany
--------------------

- [Deutsche Aidshilfe](https://www.gayhealthchat.de/)
- [makandra](https://makandra.de/)
- Deutsche Verkehrswacht (internal apps)
- GREENZERO (internal app)

**Comment by robkuz** [comment]
www.compraga-solutions.com

**Comment by ammezie** [comment]
https://mezielabs.com

**Comment by triskweline** [comment]
Mentioned on Twitter: https://melly.de/

**Comment by triskweline** [comment]
Some more URLs I found online:

- https://thenewsjunkie.com/
- https://writingourfuture.nwp.org/
- https://crushlivepoker.com/
- https://www.berghs.se/
- https://beta.card.pm/
- https://gehdenken.bastimedia.de/
- https://camelcasecollective.com/
- https://capitalvistaonline.com/
- https://board.monster/
- https://billhunt.dev/
- https://biddingrobo.com/
- https://app.bandgrid.com/
- https://github.com/Mathieu-R/streamwave-monolith
- https://github.com/edge-js/edgejs.dev

**Comment by makmic** [comment]
I found a few more:
* https://www.turquoisemountain.org/
* https://www.whitecube.com/
* https://hdontap.com/
* https://dunkerquepromotion.org/
* https://www.pantaflixgroup.com/
* https://formationarchitects.co.uk/
* https://www.ekonomism.us/

**Comment by davisums** [comment]
We're using it on www.votejar.com

Also created this website a while ago, will try to add all the links that were posted here. Feel free to add your websites too: https://hypermedia.gallery/built-with/unpoly/

**Comment by johndwells** [comment]
Hello! We at [One Darnley Road](https://www.onedarnleyroad.com/) are huge fans of Unpoly, and the agency behind a few of the sites already mentioned. Here's a full list from us:

https://crushlivepoker.com
https://www.turquoisemountain.org
https://www.whitecube.com
https://formationarchitects.co.uk
https://www.whlcarchitecture.com
https://www.ahr.co.uk
https://www.culturunners.com
https://review.gardiner.com
https://marketintel.gardiner.com
https://www.hallabintkhalid.com
https://www.jameelartshealthlab.org
https://cxtrends2023.zendesk.com
https://cxtrends.zendesk.com

:heart: 🚀

**Comment by triskweline** [comment]
- https://www.keil.arm.com/
- https://elym.green/

**Comment by triskweline** [comment]
The Urban Dictionary has been using Unpoly:
https://www.urbandictionary.com/
